### ISSUE TITLE
CC007: error report with not condition

### ISSUE THREAD

**Issue by hieulw** [apigee/apigeelint#574]
I have my Condition in flow like this and apigeelint report error while I can deploy to apigee without error
```
║ Line     │ Column   │ Type     │ Message                                                │ Rule ID              ║
╟──────────┼──────────┼──────────┼────────────────────────────────────────────────────────┼──────────────────────╢
║ 31       │ 7        │ error    │ Condition expression is invalid. Position 52,          │ CC007                ║
║          │          │          │ SyntaxError: Expected ")" or [ \t\n] but "o"           │                      ║
║          │          │          │ found.                                                 │                      ║
```
```xml
      <Condition>not (
        proxy.pathsuffix MatchesPath "**/health/*" or 
        proxy.pathsuffix MatchesPath "**/healthz/**" or 
        proxy.pathsuffix MatchesPath "**/v1/public/**" or 
      )</Condition>
```

Can you suggest me change the conditions to fix the error or should we consider this case valid in the parser.

Thank you in advance!

**Comment by hieulw** [comment]
Thank you @ssvaidyanathan, that passed linter check in my CI now.

**Comment by ssvaidyanathan** [comment]
Fixed and released in v2.74.0
